types/ember/test/debug.ts

```diff
@@ -11,7 +11,7 @@ const {
 const assert: typeof Ember.assert = Ember.assert;
 
 /**
- * @ember/debug tests
+ * `@ember/debug` tests
```

Comment:
Pending https://github.com/gajus/eslint-plugin-jsdoc/issues/1116, this seemed like the most straightforward resolution.